grr/server/grr_response_server/gui/api_plugins/timeline_test.py

```diff
@@ -206,6 +206,83 @@ def testRaisesOnIncorrectFlowType(self):
     with self.assertRaises(ValueError):
       self.handler.Handle(args)
 
+  def testBodyMultipleClients(self):
```

Comment:
Considering how similar is this test to the previous one, is there a reason why you omitted the mode?

Reply:
I've tried to be as less intrusive as possible to the existing codebase. So because initially there wasn't a filemode check in this test, I omitted it as well. I will add it. Thank you!